pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -79,7 +79,8 @@ stdenv.mkDerivation {
     ./0001-x86-Add-a-GNU_PROPERTY_X86_ISA_1_USED-note-if-needed.patch
     ./0001-x86-Properly-merge-GNU_PROPERTY_X86_ISA_1_USED.patch
     ./0001-x86-Properly-add-X86_ISA_1_NEEDED-property.patch
-  ] ++ lib.optional stdenv.targetPlatform.isiOS ./support-ios.patch;
+  ] ++ lib.optional stdenv.targetPlatform.isiOS ./support-ios.patch
+    ++ lib.optional stdenv.targetPlatform.isAarch32 ./R_ARM_COPY.patch;
```

Comment:
Could you leave a short comment above this line with a description of what this patch does, why it is needed, and when we can expect to remove it (if ever)?